subprojects/packagefiles/rdkafka/meson.build

```diff
@@ -1,14 +1,17 @@
-project('rdkafka', 'c', 'cpp', version: '1.9.2', default_options: ['cpp_std=c++17', 'c_std=c11'])
+project(
+  'rdkafka',
+  'c',
+  'cpp',
+  version: '1.9.2',
+  default_options: ['cpp_std=c++17', 'c_std=c11', 'warning_level=0'],
+  meson_version: '>=0.57.0'
```

Comment:
AFAICT the most recent feature it uses is from 0.47.0, not 57. Typo?

Reply:
```
foreach c : conf.keys()
```

0.57.0

Reply:
Hmm, that code simply isn't used at all except on Windows? What's the issue here, why isn't this applicable elsewhere, and is there an upstream bug report for it?

Reply:
yeah it's a windows only thing. There's the comment:

```
#upstream is broken and all config.h entries must be passed as compiler flags
```

What's actually happening for Windows is that all of the config options are included with a static win32_config.h. And to support dynamically setting options, it passes everything as compiler flags. No idea why this decision was made.